docs/source/model_doc/perceiver.rst

```diff
@@ -0,0 +1,228 @@
+inputs and outputs that are much larger than can be handled by standard Transformers. This means, for example,
+Perceiver IO can do BERT-style masked language modeling directly using bytes instead of tokenized inputs.
+
+The abstract from the paper is the following:
+
+*The recently-proposed Perceiver model obtains good results on several domains (images, audio, multimodal, point
+clouds) while scaling linearly in compute and memory with the input size. While the Perceiver supports many kinds of
+inputs, it can only produce very simple outputs such as class scores. Perceiver IO overcomes this limitation without
+sacrificing the original's appealing properties by learning to flexibly query the model's latent space to produce
+outputs of arbitrary size and semantics. Perceiver IO still decouples model depth from data size and still scales
+linearly with data size, but now with respect to both input and output sizes. The full Perceiver IO model achieves
+strong results on tasks with highly structured output spaces, such as natural language and visual understanding,
+StarCraft II, and multi-task and multi-modal domains. As highlights, Perceiver IO matches a Transformer-based BERT
+baseline on the GLUE language benchmark without the need for input tokenization and achieves state-of-the-art
+performance on Sintel optical flow estimation.*
+
+Here's a TLDR explaining how Perceiver works:
+
+The main problem with the self-attention mechanism of the Transformer is that the time- and memory requirements scale
+quadratically in the sequence length. Hence, models like BERT and RoBERTa are limited to a max sequence length of 512
+tokens. Perceiver aims to solve this issue by, instead of performing self-attention on the inputs, perform it on a set
+of latent variables, and only use the inputs for cross-attention. In this way, the time- and memory requirements don't
+depend on the length of the inputs anymore, as one uses a fixed amount of latent variables, like 256 or 512. These are
+randomly initialized, after which they are trained end-to-end using backpropagation.
+
+Internally, :class:`~transformers.PerceiverModel` will create the latents, which is a tensor of shape
+:obj:`(batch_size, num_latents, d_latents)`. One must provide :obj:`inputs` (which could be text, images, audio, you
+name it!) to the model, which it will use to perform cross-attention with the latents. The output of the Perceiver
+encoder is a tensor of the same shape. One can then, similar to BERT, convert the last hidden states of the latents to
+classification logits by averaging along the sequence dimension, and placing a linear layer on top of that to project
+the :obj:`d_latents` to :obj:`num_labels`.
+
+This was the idea of the original Perceiver paper. However, it could only output classification logits. In their
+follow-up work, PerceiverIO, they generalized it to let the model also produce outputs of arbitrary size. How, you
```

Comment:
In their follow-up work does not work as they is not defined. Rephrase to "In a follow-up work" for instance.